python/histogram/_units.py

```diff
@@ -1,24 +1,32 @@
 import numpy as N
 from pyre.units import *
```

Comment:
from pyre.units import * . thi can be omitted and import only the specific required functions